qemu/tests/qemu_guest_agent.py

```diff
@@ -897,6 +897,40 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_path_fsfreeze_hook(self, test, params, env):
+        """
+        Check if the default path of fsfreeze-hook is correct
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+        session = self._get_session(params, self.vm)
+        self.gagent_stop(session, self.vm)
+        error_context.context("Start gagent with -F option", logging.info)
+        params["gagent_start_cmd"] = params["gagent_start_cmd_hook"]
+        self.gagent_start(session, self.vm)
+        error_context.context("Check if default path of fsfreeze-hook is correct"
+                              " in output of 'qemu-ga --help' ", logging.info)
+        s, o = session.cmd_status_output(params["gagent_help_cmd"])
+        if "default is /etc/qemu-ga/fsfreeze-hook" in o:
```

Comment:
I think you'd better find fsfreeze-hook's full path from "rpm -ql .." and then check if the help cmd output the correct full path.
"/etc/qemu-ga/fsfreeze-hook" is a little like hardcode